code/modules/chemistry/tools/bottles.dm

```diff
@@ -198,77 +199,77 @@
 
 /obj/item/reagent_containers/glass/bottle/sulfonal
 	name = "sulfonal bottle"
-	desc = "A small bottle."
+	desc = "A small bottle of sulfonal, an extreme muscle relaxant. It has a warning label on it about impaired breathing and drowsiness."
 	bottle_style = "4"
 	amount_per_transfer_from_this = 5
 	initial_reagents = "sulfonal"
 
 /obj/item/reagent_containers/glass/bottle/synaptizine
 	name = "bottle (synaptizine)"
-	desc = "A small bottle."
+	desc = "A small bottle of synaptizine, a non-addictive stimulant whose side effects can cause regeneration of nerve cells."// pulled this one outta my ass but it's important to convey it's brain medicine
 	bottle_style = "3"
 	amount_per_transfer_from_this = 5
 	initial_reagents = "synaptizine"
 
 /obj/item/reagent_containers/glass/bottle/pancuronium
 	name = "pancuronium bottle"
-	desc = "A small bottle."
+	desc = "A small bottle of pancuronium, an extreme muslce relaxant. It has a warning label on it about impaired breathing and unconciousness."
 	bottle_style = "1"
 	amount_per_transfer_from_this = 5
 	initial_reagents = "pancuronium"
 
 /obj/item/reagent_containers/glass/bottle/antitoxin
 	name = "bottle (charcoal)"
-	desc = "A small bottle."
+	desc = "A small bottle of charcoal, a general antitoxin."
 	bottle_style = "3"
 	amount_per_transfer_from_this = 5
 	initial_reagents = "charcoal"
 
 /obj/item/reagent_containers/glass/bottle/antihistamine
 	name = "bottle (antihistamine)"
-	desc = "A small bottle of allergy medication."
+	desc = "A small bottle of antihistamines, useful in reducing allergic reactions."
 	bottle_style = "1"
 	amount_per_transfer_from_this = 5
 	initial_reagents = "antihistamine"
 
 /obj/item/reagent_containers/glass/bottle/eyedrops
 	name = "bottle (oculine)"
-	desc = "A small bottle of combined eye and ear medication."
+	desc = "A small bottle of combined eye and ear medication. A label on it reads: For best usage, apply topically."
```

Comment:
Pretty sure you can inject occuline just fine.

Reply:
you can, but since a lot of the time it's easier to splash it on, it's worth cluing players into that method of use

Reply:
i have never splashed it on lol